devicetypes/smartthings/zwave-metering-switch-secure.src/zwave-metering-switch-secure.groovy

```diff
@@ -296,19 +296,19 @@ private encapSequence(commands, delay=200) {
 private encap(physicalgraph.zwave.Command cmd) {
 	def secureClasses = [0x25, 0x5A, 0x70, 0x85, 0x8E]
```

Comment:
You can remove this.

Also, maybe there's a way to merge these two methods together now?